src/harness/util.py

```diff
@@ -285,3 +292,98 @@ def convertRequestToRequestWithCpiSignature(private_key, cpi_id,
   request['cpiSignatureData']['protectedHeader'] = jwt_message[0]
   request['cpiSignatureData']['encodedCpiSignedData'] = jwt_message[1]
   request['cpiSignatureData']['digitalSignature'] = jwt_message[2]
+
+class QueryPropagationAntennaModel:
+    nlcd_driver = nlcd.NlcdDriver()
+       
+    def getPPinPPA(self, boundarysPoints, ARCSEC):   
```

Comment:
This does not use any of the class instance variable. In fact your class does not have any instance variables (no self.xxx variables), and this method looks more like a class method.
Same for the other function `computePropagationAntennaModel` in this class which is a class method. The fact that you use self.nlcdDriver is incorrect as the nlcdDriver is a class variable, and should be access with QueryPropagationModel.nlcdDriver, and not self.nlcdDriver.

Globally what is the use of defining as a class if it does not handle internal state ?? 
I recommend that we move these 2 functions elsewhere as just routines:
 - getPPinPPA : into the reference_models/ppa/  directory. It will address Kate remark as well.
 - computePropagationAntennaModel can status in this util file, as it is very specific to the test.